app/views/trustee/MaxedOutTrusteesView.scala.html

```diff
@@ -28,30 +33,32 @@
     heading: String)(implicit request: Request[_], messages: Messages)
 
 @main_template(
-    title = heading
+    title = heading,
+    showBackLink = true
 ) {
 
     @formHelper(action = AddATrusteeController.submitComplete(), 'autoComplete -> "off") {
 
-        @components.back_link()
+        @headingComp(heading)
 
-        @components.heading(heading)
+        @addToList(inProgressTrustees, "inprogress", "trustee")
 
-        @components.data_list(inProgressTrustees, "inprogress", "trustee")
-
-        @components.data_list(completeTrustees, "complete", "trustee")
+        @addToList(completeTrustees, "complete", "trustee")
 
         @if(!isLeadTrusteeDefined) {
-            @components.inset_text("addATrustee.lead-trustee.required", "addATrustee")
+            <div id="inset-text--addATrustee" class="indent">
+                @messages("addATrustee.lead-trustee.required")
+            </div>
         }
 
         <p>@messages("addATrustee.p1") <a href="@controllers.routes.ReplacingLeadTrusteeController.onPageLoad().url">@messages("addATrustee.p1.a")</a>.</p>
```

Comment:
This link is is missing the govuk-link class so it should be `<a class="govuk-link" href=`

Spotted on this view but may be other <a> tags missing it

Reply:
Update anywhere the I found <a href=.

Reply:
Sorry, just seen on a UTR page, think trustees/org `<a target="_blank" href="https://www.gov.uk/find-lost-utr-number">find a lost UTR number (opens in a new window or tab)</a>` which will have been missed by your `<a href=` search